src/Backend/Form/Type/FileType.php

```diff
@@ -134,10 +134,20 @@ public function configureOptions(OptionsResolver $resolver)
             [
                 'data_class' => AbstractFile::class,
                 'empty_data' => function () {
-                    $emptyData = new StdClass();
-                    $emptyData->file = null;
+                    return new class extends StdClass {
```

Comment:
true, but that is because they are rather alike, but still, I wouldn't bring it outside since it is a pure implementation detail